proto/bragi/bragi_service.proto

```diff
@@ -27,8 +28,12 @@ service Bragi {
 
 message MatchTimelineRequest {}
 
-message MatchTimelineResponse {
-  repeated Match matches = 1;
+message MatchTimelineMessage {
+  oneof message {
```

Comment:
Suggestion: rename message -> payload ("payload"  name is used for such cases in bragi schema)

Reply:
Done